### DISCUSSION TITLE
Compute moments on latent space with treatment groups

### DISCUSSION THREAD

**Discussion by flde** [theislab/scvelo#842] [Q&A]
Hi, many thanks for the great tools and documentation! 

I am working with cells that were stimulated. To integrate them I used the scVI VAE and constructed a kNN on the latent space. I run RNA velocity on all samples combined on the latent kNN but used a 0.1 min_likelihood threshold to compute the velocities (dynamic). I assume that genes with a high likelihood across treatment groups are canonical to the overall biological process. Next I check for genes with differential kinetics to determine treatment effects. 

However, when computing the first/second order moments on the latent space derived kNN the imputation is potentially done on cells across different treatment groups, right? Which I assume is not correct to do? Would it make sense to compute the moments separately for each group and then combine the M matrices for combined downstream analysis? 

Many thanks!

**Comment by WeilerP** [comment]
@flde, sorry, I might misunderstand: It sounds like you do not expect development across/between treatment groups. If this is the case, why try inferring RNA velocity on the entire dataset instead of for each group individually?
If you do expect development across/between, I do not see why imputing the entire dataset in one go poses a problem. Moments are based on transcriptomic similarity, _i.e._, even if you impute on a set of cells from different treatment groups these must be similar on a transcriptomic level (otherwise they wouldn't in each other's neighbors).
